source/triggers/aws-eventbridge.txt

```diff
@@ -152,13 +171,363 @@ Procedure
+
+            A draft represents a group of application changes that you
+            can deploy or discard as a single unit. If you don't create
+            a draft, updates automatically deploy individually.
+
+            To create a draft, send a ``POST`` request with no body to
+            the :admin-api-endpoint:`Create a Deployment Draft
+            <tag/deploy/operation/adminCreateDeploymentDraft>` endpoint:
+
+            .. code-block:: bash
+
+               curl -X POST 'https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/drafts' \
+               -H 'Content-Type: application/json' \
+               -H 'Authorization: Bearer <access_token>'
+         
+         .. step:: Create the Error Handler Function
+
+            Create the function to handle errors for a failed AWS 
+            EventBridge trigger via a ``POST`` request to the 
+            :admin-api-endpoint:`Create a new
+            Function <operation/adminCreateFunction>` endpoint.
+
+            .. code-block:: bash
+
+               curl -X POST \
+                  https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/functions \
+                  -H 'Authorization: Bearer <access_token>' \
+                  -d '{
+                     "name": "string",
+                     "private": true,
+                     "source": "string",
+                     "run_as_system": true
+                     }'      
+                     
+         .. step:: Create the AWS EventBridge Trigger
+
+            Create the AWS EventBridge Trigger with error handling 
+            enabled via a ``POST`` request to the 
+            :admin-api-endpoint:`Create a Trigger <operation/
+            adminCreateTrigger>` endpoint.
+
+            .. code-block:: bash
+
+               curl -X POST \
+                  https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/triggers \
+                  -H 'Authorization: Bearer <access_token>' \
+                  -d '{
+                     "name": "string",
+                     "type": "DATABASE",
+                     "config": [
+                        "service_id": "string",
+                        "database": "string",
+                        "collection": "string",
+                        "operation_types": {
+                           "string"
+                        },
+                        "match": ,
+                        "full_document": false,
+                        "full_document_before_change": false,
+                        "unordered": true
+                     ],
+                     "event_processors": [
+                        "AWS_EVENTBRIDGE": {
+                           "account_id": "string",
+                           "region": "string",
+                           "extended_json_enabled": false
+                        },
+                     ],
+                     "error_handler": [
```

Comment:
`error_handler` should have curly braces rather than square brackets around the data